website/test/fixtures/external-code-embed/QueryOptions.txt

```diff
@@ -32,7 +32,7 @@ export interface QueryOptions {
    * Determines if column values should be converted to native JavaScript types. It is not recommended (and may go away / change in the future)
    * to disable type casting, but you can currently do so on either the connection or query level. (Default: true)
    *
-   * You can also specify a function (field: any, next: () => void) => {} to do the type casting yourself.
+   * You can also specify a function (field: any, next: () => unknown) => {} to do the type casting yourself.
```

Comment:
There is no need to change the test fixtures or resources in `./website`. They just ensure string extractions in the `ExternalCodeEmbed` tests and are for internal use only 🙋🏻‍♂️